Source/AwsCommonRuntimeKit/auth/credentials/AwsCredentials.swift

```diff
@@ -2,15 +2,13 @@
 //  SPDX-License-Identifier: Apache-2.0.
 
 import AwsCAuth
+import Foundation
 
-public final class CRTCredentials {
+public class AwsCredentials {
```

Comment:
Likely not for this PR as i imagine there are other places we'll want to do this, but in Swift it'd be preferable to name this `AWSCredentials`

Reply:
Can we keep the `final` designation? I don't think we want folks to subclass this and if it weren't for the need to do some manual cleanup in the `deinit` this would likely be a `struct` instead of a `class` anyways.

Reply:
There is so much inconsistency in this codebase with how acronyms are capitalized in CamelCase words:
AwsCredentials, AWSString, CRTIMDSClient, HttpClientConnection

We should absolutely do a capitalization consistency pass in one big PR.

The various aws-crt-XYZ language bindings *generally* lean towards only capitalizing the first letter in a 3+ letter acronym. But if Swift leans strongly towards capitalizing every letter, we should do what's common in Swift-land

Reply:
I agree one big PR would be preferable for all the names changes. Our other Swift SDK repos (smithy-swift and aws-sdk-swift) also are very inconsistent. I'll file an issue in aws-sdk-swift if there isn't one already, to make this consistent across our Swift sdks.

@waahm7 Let's prefer to always use `final` unless we are subclassing the class internally or if we want to allow consumers to subclass (which you're correct, will require the `open` keyword).